README.md

```diff
@@ -55,7 +70,18 @@ Create a new file in your project, `address.js`.
 
 ```javascript
 const { default: Resolution } = require('@unstoppabledomains/resolution');
-const resolution = new Resolution();
+// Obtained by creating an account with Infura
+const ethProviderURL = `https://mainnet.infura.io/v3/${infuraProjectId}`;
+const resolution = new Resolution({
+  cns: {
+    url: ethProviderURL,
+    network: 'mainnet',
+  },
+  ens: {
+    url: ethProviderURL,
+    network: 'mainnet',
+  },
+});
```

Comment:
We can also mention it in a comment that if a user doesn't pass the configuration object, we'll use https://main-rpc.linkpool.io by default.